.github/workflows/ci-build-binaries.yml

```diff
@@ -57,10 +61,6 @@ jobs:
   build:
     name: ${{ inputs.os }}-${{ inputs.arch }}
     runs-on: ${{ inputs.build-platform }}
-
-    permissions:
-      id-token: write
```

Comment:
TODO: move this to a separate PR

This was added in https://github.com/pulumi/pulumi/pull/11310, but I don't *think* this is needed